CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ You can find its changes [documented below](#032-2025-09-10).
 
 This release has an [MSRV][] of 1.82.
 
+### Changed
+
+* Skip serializing/deserializing `PhantomData` marker fields of `OpaqueColor`, `AlphaColor`, and `PremulColor` when the `serde` feature is enabled.
```

Comment:
This needs the PR and author links (don't forget to also add these to the link reference definitions at the bottom)